lib/restaurant.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'order'
+require_relative 'menu'
+
+class Restaurant
+  attr_reader :order
+
+  def initialize(menu, messenger = nil)
+    @menu = menu
+    @menu.read_menu
+    @messenger = messenger
+  end
+
+  def see_menu
+    # TODO: how do I test this?
+    puts @menu
+  end
+
+  def add_to_order(dish, quantity = 1)
+    if @order.nil?
```

Comment:
Could `@order` be created in `initialize` to avoid having to write this extra code?